pkg/server/controllers.go

```diff
@@ -1238,6 +1239,52 @@ func (s *Server) installReplicationController(ctx context.Context, config *rest.
 	})
 }
 
+func (s *Server) installGarbageCollectorController(ctx context.Context, config *rest.Config, server *genericapiserver.GenericAPIServer) error {
+	config = rest.CopyConfig(config)
+	config = rest.AddUserAgent(config, garbagecollector.ControllerName)
+
+	kubeClusterClient, err := kcpkubernetesclientset.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	metadataClient, err := kcpmetadata.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	// TODO: make it configurable
```

Comment:
Do we want to bubble this up through options in this PR?

Reply:
Ok not to - same as quota